src/net/tls.cc

```diff
@@ -1367,6 +1372,7 @@ class session : public enable_lw_shared_from_this<session> {
             _output_pending = _out.put(std::move(msg).release());
             return n;
```

Comment:
So we change the meaning of this code.

Reply:
I'm not sure what you refer to that changes the meaning of this code.
Did you refer to the `get_exception()` call above (that I agree needs to be fixed)
or the catch(std::system_error&) below?

Reply:
The fact that get_exception() alters _output_pending.